notebooks/official/generative_ai/text_embedding.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<p>You can clear this output as it doesn't add much value to the reader.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done!</p>